ext/lmdb-safe/lmdb-safe.hh

```diff
@@ -1,4 +1,5 @@
 #pragma once
+#include <string_view>
```

Comment:
this commit seems totally unrelated here.

Reply:
It was to fix an error message I was getting in my IDE, so it's part of "cleanup". Is it OK if I keep it?

Reply:
this can stay